samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java

```diff
@@ -138,8 +140,8 @@ public void processAction(ActionEvent ae) throws
       // 1. get managed bean
       DeliveryBean delivery = (DeliveryBean) ContextUtil.lookupBean("delivery");
 
-      delivery.setLocale(preferencesService.getLocale(AgentFacade.getAgentString()));
-      
+      delivery.setLocale(preferencesService.getLocale(userDirectoryService.getCurrentUser().getId()));
```

Comment:
Should this be using the agent?

Reply:
Why?

Reply:
I used agent in the first place, just because it was there already.

Reply:
I suppose it could. I'll try it out now.

Reply:
I switched them because I was testing if an instructor didn't have an agent string, like a student. That was it.

Reply:
Sorted.